samples/benchmark/bunnymark/kotlin/src/main/kotlin/godot/samples/benchmark/bunnymark/BunnymarkV1DrawTexture.kt

```diff
@@ -0,0 +1,97 @@
+package godot.samples.benchmark.bunnymark
+
+import godot.core.*
+import godot.*
+import org.godotengine.kotlin.annotation.RegisterClass
+import org.godotengine.kotlin.annotation.RegisterFunction
+
+@RegisterClass("benchmarks/BunnymarkV1DrawTexture/kot")
+class BunnymarkV1DrawTexture: Node2D() {
+
+    data class Bunny(var position:Vector2, var speed: Vector2)
+
+    private var bunnies = ArrayList<Bunny>()
+    private var grav = 500
+    private var bunnyTexture = Texture from ResourceLoader.load("res://images/godot_bunny.png")
+    private val rng = RandomNumberGenerator()
```

Comment:
Variable name is not self explanatory. I would prefer `randomNumberGenerator`.
But you can leave it if you prefer this.